site/src/pages/ManagementSettingsPage/OrganizationProvisionersPageView.tsx

```diff
@@ -32,8 +40,28 @@ export const OrganizationProvisionersPageView: FC<
 				<PageHeaderTitle>Provisioners</PageHeaderTitle>
 			</PageHeader>
 			<Stack spacing={4.5}>
-				{provisioners.map((provisioner) => (
-					<Provisioner key={provisioner.id} provisioner={provisioner} />
+				{provisioners.builtin.length > 0 && (
+					<ProvisionerGroup
+						buildInfo={buildInfo}
+						type="builtin"
+						provisioners={provisioners.builtin}
+					/>
+				)}
+				{provisioners.psk.length > 0 && (
+					<ProvisionerGroup
+						buildInfo={buildInfo}
+						type="psk"
+						provisioners={provisioners.psk}
+					/>
+				)}
+				{[...provisioners.keys].map(([keyId, provisioners]) => (
```

Comment:
Why are we spreading here if `.map` always makes a new array?

Reply:
because `provisioners.keys` is a `Map<string, ProvisionerDaemon[]>`, which does not have a `map` method (yet)